OJS.Workers.ExecutionStrategies/RubyExecutionStrategy.cs

```diff
@@ -26,11 +29,11 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
             var executor = new RestrictedProcessExecutor(this.BaseTimeUsed, this.BaseMemoryUsed);
             var checker = Checker.CreateChecker(
```

Comment:
`executor` and `checker` seem to be defined almost everywhere, identically. Have you considered moving their definition to `ExecutionStrategy` and exposing them as properties, to reduce the boilerplate?